ansible-ipi-install/inventory/hosts

```diff
@@ -0,0 +1,11 @@
+[masters]
+master-0 name=master-0 role=master user=admin password=password ipmi_address=192.168.1.1 mac=ec:f4:bb:da:0c:58
+master-1 name=master-1 role=master user=admin password=password ipmi_address=192.168.1.2 mac=ec:f4:bb:da:32:88
+master-2 name=master-2 role=master user=admin password=password ipmi_address=192.168.1.3 mac=ec:f4:bb:da:0d:98
+
+[workers]
+worker-0 name=worker-0 role=worker user=admin password=password ipmi_address=192.168.1.4 mac=ec:f4:bb:da:0c:18
+worker-1 name=worker-1 role=worker user=admin password=password ipmi_address=192.168.1.5 mac=ec:f4:bb:da:32:28
+
+[provisioner]
+provisioner.example.com
```

Comment:
Well you definitely need the FQDN for the provision host as your playbook is connecting here. Regarding masters and workers it doesn't matter because the information gathered in the inventory/hosts file is used to fill out the appropriate information in the install-config.j2 template file

Reply:
You are correct that nothing in the playbooks ever directly executes on the master/worker nodes. However, we use this info to generate the install-config.yaml file. Since we are still talking about an inventory of hosts, does it not make more sense to keep it there versus as variables? If we decide to make them vars, do you for see then making a list for each item? i.e. ipmi_address, user, password, name etc?

Reply:
the only reason I considered them as vars is it centralizes everything related to a cluster in a Vars file, while the provisioner might be able to execute multiple cluster setups.

>  If we decide to make them vars, do you for see then making a list for each item? i.e. ipmi_address, user, password, name etc?

It's something worth experimenting with to see what formatting feels most natural. I'm not 100% certain what the constraints are in expressing something like this as vars.

Reply:
@davidvossel I think its better to have them in the inventory file to be honest. Also speaking to @dcritch , I'm going to make some changes how I handle vars. Instead of having a group_vars/all file, I'm going to place all that info side a sample inventory file. This way people can use different inventory files to install environments without the need to change the playbook.yml